README.md

```diff
@@ -114,6 +114,25 @@ npm run dev
 
 > Note: `frontend/src/config.ts` contains `BACKEND_URL`. If you need your frontend to point to local backend server, uncomment `export const BACKEND_URL = "http://localhost:8787"`. 
 
+
+#### Running Frontend and Backend Concurrently
+
+To make the developer experience smoother, you can now run both the frontend and backend simultaneously using a single command from the project root.
+
+##### Steps:
+
+- Ensure you have project root folder. install packages with
+   ```sh
+   npm install
```

Comment:
Shouldn't npm install also install the packages for frontend and backend ?

Reply:
Running `npm install` at the root installs only the root-level dependencies, such as `concurrently`. You need to run npm install separately in both the frontend and backend directories to install their respective dependencies.

Reply:
Please check is its possible to run npm install from the root so that it does all of the following with 1 command.

- install concurrently at the root
- cd frontend && npm install
- cd backend && npm install

Reply:
To address the concern that npm install should install packages for both the frontend and backend, we can create an additional script in the root `package.json` to handle this:

```json
"scripts": {
   "install:all": "concurrently --prefix \"[{name}]\" --names \"backend-install,frontend-install\" -c \"blue,green\" \"cd backend && npm install \" \"cd frontend && npm install\" ",
   "dev": "concurrently --prefix \"[{name}]\" --names \"backend,frontend\" -c \"blue,green\" \"cd backend && npm run dev \" \"cd frontend && npm run dev\" "
}
```
With this addition, running `npm run install:all` will install all dependencies for both the backend and frontend: